src/transformers/generation_utils.py

```diff
@@ -483,31 +483,6 @@ def _expand_inputs_for_generation(
             model_kwargs["encoder_outputs"] = encoder_outputs
         return input_ids, model_kwargs
 
-    @staticmethod
-    def _init_sequence_length_for_generation(
```

Comment:
This function is only used twice and essentially boils down to a single line -> therefore it doesn't not merit to be factored out into a function.